projects/templates/projects/chameleon_used_in_research.html

```diff
@@ -0,0 +1,38 @@
+{% extends "layout/default.html" %}
+{% load static bootstrap3 chameleon_tags %}
+{% block title %}Add Publications for Project - {% endblock %}
+{% block content %}
+
+  <h1>Chameleon Used In Research</h1>
+  <table id="example" class="table table-striped table-bordered align-middle" style="width:100%">
+    <thead>
+        <tr>
+            <th>Title</th>
+            <th>Year</th>
+            <th>Forum</th>
+            <th>Authors</th>
+            <th>Cites</th>
```

Comment:
This should be "Cited by" I think, like how google scholar displays it. "Cites" is unclear whether that is how many papers this article cites, or how many papers cite this article.